readthedocs/core/management/commands/update_api.py

```diff
@@ -14,10 +14,15 @@ class Command(BaseCommand):
     the site. Invoked via ``./manage.py update_repos``.
     """
 
+    def add_arguments(self, parser):
+        parser.add_argument('--docker', action='store_true', default=False)
+
     def handle(self, *args, **options):
+        docker = options.get('docker', False)
         if len(args):
             for slug in args:
                 project_data = api.project(slug).get()
                 p = tasks.make_api_project(project_data)
                 log.info("Building %s" % p)
-                tasks.update_docs(pk=p.pk)
+                update_docs = tasks.UpdateDocsTask()
```

Comment:
Is there a reason we don't just put this in the tasks.py so it's importable like previously?